tailwind.config.js

```diff
@@ -2,7 +2,20 @@
 module.exports = {
   content: ["./src/**/*.{html,js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+          brand: {
+            black: "#000F1C",
+            dark: "#0B1828",
+            DEFAULT: "#50A6D6",
+            grey: "#475466",
+            white: "#FFFFFF",
```

Comment:
don't just call them by their plain names, try naming them based on what they recommend from the official tailwind docs [here]<https://tailwindcss.com/docs/customizing-colors#naming-your-colors> or  [this]<https://medium.com/@stefanmorris/proper-naming-of-color-variables-683264fec0f4>.